extension/contentfragment/content/src/content/jcr_root/apps/core/wcm/extension/components/contentfragment/v1/contentfragment/clientlibs/editor/authoring/js/editAction.js

```diff
@@ -37,8 +37,13 @@
             // get the path of the content fragment
             var fragmentPath = $(editable.dom).find("." + CLASS_CONTENTFRAGMENT).attr(ATTRIBUTE_PATH);
             if (fragmentPath) {
+                var fragmentEditUrl = EDITOR_URL + fragmentPath;
+                var fragment = Granite.author.CFM.Fragments.adaptToFragment(editable.dom);
```

Comment:
@josephrignanese - `Granite.author` is available in the closure as `ns` so we could use `ns.CFM.Fragments.adaptToFragment` just to be consistent with other usages.